modules/gateway/gateway.go

```diff
@@ -343,5 +345,36 @@ func New(addr string, bootstrap bool, persistDir string, bcInfo types.Blockchain
 	return g, nil
 }
 
+func (g *Gateway) connectToBootstapPeers(closeChan chan struct{}, bootstrapPeers []modules.NetAddress) {
+	defer close(closeChan)
+
```

Comment:
```suggestion
g.startConnectingToBootstrapPeers(bootstrapPeers)
```

Add method call before loop so we immediately start to connect